openfga/v1/openfga_service.proto

```diff
@@ -1561,7 +1561,7 @@ message Assertion {
     (google.api.field_behavior) = REQUIRED
   ];
 
-  openfga.v1.ContextualTupleKeys contextual_tuples = 3 [json_name = "contextual_tuples"];
+  repeated TupleKey contextual_tuples = 3 [json_name = "contextual_tuples"];
```

Comment:
We're missing the limits on this like in https://github.com/openfga/api/blob/591c8fa7765ca7b10feec619cf7d849380da3757/openfga/v1/openfga.proto#L229-L230

The max number of contextual tuples should be 20. We need the two annotations referenced above for this.